docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: '3.8'
+services:
+  app:
+    build:
+      context: .
+      args:
+        NODE_ENV: development
+    volumes:
+      - node_modules:/app/node_modules
+      - dist:/app/dist
+      - .:/app:delegated
```

Comment:
When you bind a host path to the container, it will prefer the host, including non existent paths. So if your container expects a path /app/dist to exist... but it doesn't exist on the host when you start the container, it will actually remove the directory.... not totally sane but whatever.

This is solved, actually, by using the named volumes, but by adding "delegated" we are making a feeble attempt to instruct docker to preserve container created files even if they don't exist on the host.